diesel/src/upsert/on_conflict_extension.rs

```diff
@@ -53,14 +53,15 @@ where
     /// #     let conn = establish_connection();
     /// #     #[cfg(feature = "postgres")]
     /// #     conn.execute("TRUNCATE TABLE users").unwrap();
-    /// #     #[cfg(feature = "sqlite")]
-    /// #     conn.execute("DELETE FROM users").unwrap();
+    /// #[cfg(feature = "postgres")]
```

Comment:
I think that these how to use cfg attr is bad, but I don't know how to write better especially features documentation test.

Reply:
Maybe move those examples to a postgres specific section  (== below a headline/something indicating that this is only for postgres) of this doc comment?
For the cfg attr usage: This is also done in other places, so this is ok: https://github.com/diesel-rs/diesel/blob/2ce0e4ea0fda474459139042247512f0c8b254cf/diesel/src/query_builder/sql_query.rs#L62

Reply:
Reorder the doc tests to indicate what's only working on postgres, and then hide/remove the cfg attribute.